learning_journal/models/mymodel.py

```diff
@@ -8,11 +11,12 @@
 from .meta import Base
 
 
-class MyModel(Base):
-    __tablename__ = 'models'
+class Entry(Base):
+    __tablename__ = 'entries'
     id = Column(Integer, primary_key=True)
-    name = Column(Unicode)
-    value = Column(Integer)
+    title = Column(Unicode)
+    # creation_date = Column(DateTime, default=datetime.date)
```

Comment:
Why is this line commented out?  Seems like it might be a useful field to sort posts by / display when a post was made